### DISCUSSION TITLE
How to add ZCU104 board support to HLS4ML?

### DISCUSSION THREAD

**Discussion by AnouarITI** [fastmachinelearning/hls4ml#726] [Q&A]
Hello 

I want to add the board support for ZCU104 to the Vivado backend, I made the following steps:
1- Added the necessary lines similarly to the zcu102 in `supported_boards.json` file
2- Added a folder to `hls4ml/templates/vivado_accelerator`, named it zcu104, where I have the python driver folder and the tcl scripts required to build a project for the zcu104.
3- I run `python pip install .` to reinstall hls4ml with the made modification.

However, I still see that the board is not supported.
I am missing something here.

**Comment by sandeep1404** [comment]
Hi, did you resolve the issue, even I have the same doubt, can you help me if you resolve the issue.

**Comment by jmduarte** [comment]
@AnouarITI @sandeep1404 I started a PR #752 to add the ZCU104 board, but please feel free to make suggestions for the tcl file, which may not be totally correct yet.
